.github/ISSUE_TEMPLATE/improve-the-site.yml

```diff
@@ -0,0 +1,48 @@
+name: Improve the docs.github.com site
+description: Make a suggestion or report a problem about the technical implementation of docs.github.com.
+title: "'[What I really want the title to be]': "
+labels:
+  - content
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **HUBBERS!! THIS IS GITHUB/DOCS OPEN SOURCE REPO. OPEN AN ISSUE IN GITHUB/DOCS-CONTENT INSTEAD.**
```

Comment:
Any changes you make to this line or additional `.` you add to the template above ☝️ should be added to this one too, to keep them matchy matchy 🤝